ros_holographic/nodes/visual_memory_module.py

```diff
@@ -101,6 +101,31 @@ def handle_label_dump(self, req):
       print "Dumped vector to %s" % path
       return self.handle_label_probe(req)
 
+  def handle_dump(self, req):
+      if self.m is None:
+        print "Memory is empty"
+        return False
+      if req.Path is '':
+        timestamp = time.strftime("%Y-%m-%d-%H:%M:%S")
+        path = '/tmp/%s.vec' % timestamp
```

Comment:
You can use the tempfile module for that: 
https://docs.python.org/2/library/tempfile.html

it ships by default